mods/braindance_protocol/cheats/teleport.lua

```diff
@@ -48,14 +48,14 @@ function Teleport.GenericTotentanzClub()
     Utilities.FinishProtocol(moduleName)
 end
 
-function Teleport.GenericDMCInside()
+function Teleport.GenericDeMCInside()
     local moduleName = "Teleporting to Dark Matter Club (inside)"
     Utilities.StartProtocol(moduleName)
     Game.TeleportPlayerToPosition("-346.081238","221.274918","191.086502")
     Utilities.FinishProtocol(moduleName)
 end
 
-function Teleport.GenericDMCOutside()
+function Teleport.GenericDeMCOutside()
```

Comment:
Is this a typo? Why DeMC instead of DMC if it is for Dark Matter Club ?

Reply:
Oh, whoops. Should've been DaMC or DMC. I'll fix that 👍